apps/carbon/app/components/Form/Color.tsx

```diff
@@ -35,7 +35,10 @@ const ColorPicker = ({ name, label }: ColorFieldProps) => {
           <button
```

Comment:
```suggestion
          <button
            type="button"
```

This is a suggestion from @sapter -- basically any button in a form that does not have a type specified will default to a submit, if I understand correctly.